src/Nancy/DefaultRuntimeEnvironmentInformation.cs

```diff
@@ -35,10 +36,20 @@ private static bool GetDebugMode(ITypeCatalog typeCatalog)
             {
                 var assembliesInDebug = typeCatalog
                     .GetTypesAssignableTo<INancyModule>(TypeResolveStrategies.ExcludeNancy)
+#if DOTNET5_4
+                    .Select(x => x.GetTypeInfo().Assembly.GetCustomAttributes(typeof(DebuggableAttribute)))
+                    .Where(x => x.Any());
+
+                    return assembliesInDebug.Any(d => ((DebuggableAttribute)d.First()).IsJITTrackingEnabled);
+#else
                     .Select(x => x.Assembly.GetCustomAttributes(typeof(DebuggableAttribute), true))
                     .Where(x => x.Length != 0);
+                    
+                    return assembliesInDebug.Any(d => ((DebuggableAttribute)d[0]).IsJITTrackingEnabled);

```

Comment:
Has the return of `GetCustomAttributes` changed from? Looks like this could be made much more cross-platform if it's only the `GetTypeInfo()` that is required on `DOTNET5_4` since it works on both platforms?